src/main/java/org/mockito/internal/framework/DefaultMockitoFramework.java

```diff
@@ -23,4 +24,24 @@ public MockitoFramework removeListener(MockitoListener listener) {
         mockingProgress().removeListener(listener);
         return this;
     }
+
+    /**
+     * Adds listener only if it is an instance of MockitoListener.
+     * Useful to avoid casting in internal code.
+     */
+    public static void tryAddListner(Object listener) {
```

Comment:
@szczepiq This smells, it looks like a design problem. Why don't you let MockitoTestListener  extend MockitoListener?